portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -22,28 +22,27 @@
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet takes in hard-coded comments **/
 @WebServlet("/data")
 public final class DataServlet extends HttpServlet {
 
-	@Override
-    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        
-        PortfolioComments commentsRecord = new PortfolioComments();
-
-        // Add Sample Comments
-        commentsRecord.addComment("You rock! Keep on coding!");
-        commentsRecord.addComment("This is momma! You got this girl :-)");
-        commentsRecord.addComment("This is dad! Go break some eggs, babygirl.");
-
-        // TODO: Put Each Comment in JSON Form and Print it
-
-    }
-
-    /**
-    * Converts the commentsRecord instance into a JSON string using manual String concatentation.
-    */
-    private String convertToJson(PortfolioComments commentsRecord) {
-        // TODO: Form String in JSON form using String concatentation
-    }
-}
\ No newline at end of file
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  
+  PortfolioComments commentsRecord = new PortfolioComments();
+
```

Comment:
nit: join variable declaration with logic section. AKA, no need for newline here.